docs/src/dev-guide/design-kv-ir-streams/background.md

```
@@ -0,0 +1,318 @@
+  timersObj --> timersStage2Null
+  timersObj --> timersStage2Float
+:::
++++
+**Figure 3**: The archive's schema tree after adding the log events from [Figure 1](#figure-1). Each
+node's label is of the form `<ID> <key>: <type>`.
+::::
+<!-- markdownlint-enable MD013 -->
+
+### Encoding & storing event values
+
+For each log event, clp-s uses a variety of encoding methods to compactly encode its values before
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 98

(MD013, line-length)